doc/2/guides/introduction/2-general-purpose-backend/index.md

```diff
@@ -0,0 +1,204 @@
+---
+code: false
+type: page
+title: General Purpose Backend
+description: Kuzzle main concepts and features overview
+order: 200
+---
+
+# General Purpose Backend
+
+Kuzzle is a generic backend offering **the basic building blocks common to every application**.
+
+Rather than redeveloping functionalities that **do not bring value for your users**, Kuzzle proposes to use its basic building blocs in order to **build high-level business features**.
```

Comment:
```suggestion
Rather than redeveloping functionalities that **do not bring value for your users**, Kuzzle proposes to use its basic building blocks in order to **build high-level business features**.
```